Assets/Scripts/Waypoint Scripts/WaypointProperties.cs

```diff
@@ -37,7 +37,10 @@ public class WaypointProperties : MonoBehaviour
         private LineRenderer LineProperties;
         private CapsuleCollider lineCollider;
 
-        private GameObject world;
+        private GameObject World;
+        private Vector3 WorldScaleInitial;
```

Comment:
what are all the scales for? I believe earlier there was another file with similar scaling variables, it it possible to just all reference worldproperties or smtg

Reply:
another pr will be made for this